dev/tests/api-functional/testsuite/Magento/GraphQl/ConfigurableProduct/AddConfigurableProductToCartTest.php

```diff
@@ -92,6 +93,58 @@ public function testAddProductIfQuantityIsNotAvailable()
         $this->graphQlMutation($query);
     }
 
+    /**
+     * @magentoApiDataFixture Magento/ConfigurableProduct/_files/product_configurable_sku_with_disabled_simple.php
+     * @magentoApiDataFixture Magento/Checkout/_files/active_quote.php
+     * @expectedException Exception
+     * @expectedExceptionMessage The product with SKU "simple_20" is disabled.
+     */
+    public function testAddProductIfVariationIsDisabled()
+    {
+        $searchResponse = $this->graphQlQuery($this->getFetchProductQuery('configurable'));
+        $product = current($searchResponse['products']['items']);
+
+        $maskedQuoteId = $this->getMaskedQuoteIdByReservedOrderId->execute('test_order_1');
+        $parentSku = $product['sku'];
+        $sku = 'simple_20';
+
+        $query = $this->getQuery(
+            $maskedQuoteId,
+            $parentSku,
+            $sku,
+            1
+        );
+
+        $this->graphQlMutation($query);
+    }
+
+    /**
+     * @magentoApiDataFixture Magento/ConfigurableProduct/_files/product_configurable_sku_without_stock.php
+     * @magentoApiDataFixture Magento/Checkout/_files/active_quote.php
+     * @expectedException Exception
+     */
```

Comment:
Why not use?
```php
/**
* @expectedExceptionMessage Could not add the product with SKU configurable to the shopping cart: This product is out of stock.
*/
```

Reply:
Because of chars limit by line, it exceeded the number of chars, so I have to put it as:
$this->expectExceptionMessage(
            "Could not add the product with SKU configurable to the shopping cart: This product is out of stock."
        );